lib/ReactViews/Notification/HelpScreenWindow.jsx

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+import ObserveModelMixin from '../ObserveModelMixin';
+import React from 'react';
+import parseCustomHtmlToReact from '../Custom/parseCustomHtmlToReact';
+import Styles from './help-screen-window.scss';
+import classNames from 'classnames';
+
+const HelpScreenWindow = React.createClass({
+    mixins: [ObserveModelMixin],
+
+    propTypes: {
+        terria: React.PropTypes.object,
+        viewState: React.PropTypes.object
+    },
+
+    render() {
+        const helpScreen = this.props.terria.helpScreen && this.props.terria.helpScreen[this.props.terria.helpScreen.length - 1];
+        const windowClass = classNames(Styles.window, {
+            [Styles.isActive]: helpScreen
+        });
+        const buttonText = helpScreen && helpScreen.totalNumberOfScreens === helpScreen.currentScreenNumber ? 'DONE' : 'NEXT';
+        const positionLeft = helpScreen && helpScreen.left;
+        const positionTop = helpScreen && helpScreen.top;
```

Comment:
It's in SOE. The help is specific to SOE, so I thought I should only use the help screens in SOE.